modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,174 @@
+* The LiveCD operating system image, which includes the `pre-configured ``salt-minion`` service.
+For the image, see https://build.opensuse.org/package/show/systemsmanagement:yomi/openSUSE-Tumbleweed-Yomi
+
+
+To use Yomi for installing your client operating system, you need to follow this process:
+
+* Install and configure the ``salt-master``service.
+* Install the Yomi formula package.
+* Prepare the Salt pillar for the new installation.
+* Boot the new systems with the operating system image, or use PXE boot.
+
+For more information about using Yomi, see https://github.com/openSUSE/yomi/blob/master/README.md
+
+
+
+== Install and Configure salt-master
+
+You will require the ``salt-master`` service to be able to use Yomi for your client operating system.
+If you have not yet installed it on the {productname} Server, you will need to do this step first.
+
+
+
+.Procedure: Installing salt-master
+
+. On the {productname} Server, at the command prompt, as root, install the ``salt-master`` service:
++
+----
+zypper in salt-master
+----
+. Use ``systemctl`` to enable the service:
++
+----
+systemctl enable --now salt-master.service
+----
+
+Yomi uses a Salt pillar to provide configuration information for the installation.
+This can include the layout of the hard disks, the software
+patterns to install, or the users to create.
+
+By default, Salt searches for states in the [path]``/srv/salt/`` directory.
+By default, Salt searches for pillars in [path]``/srv/pillar``.
+
+You do not need to perform this step, if you are using the Yomi formula.
+
+
+.Procedure: Configuring salt-master
+
+. On the {productname} Server, at the command prompt, as root, create two new directories for the configuration file:
++
+----
+mkdir -p venv/etc/salt/pki/{master,minion}
+----
+.  Create two new virtual environments:
++
+----
+venv/etc/salt/autosign_grains
+venv/var/cache/salt/master/file_lists/roots
+----
+. Add the configuration information to the new location:
++
+----
+cat <<EOF > venv/etc/salt/master
+root_dir: $(pwd)/venv
+
+file_roots:
+  base:
+    - $(pwd)/srv/salt
+
+pillar_roots:
+  base:
+    - $(pwd)/srv/pillar
+EOF
+----
+
+
+
+== Install the Yomi Formula Package
+
+
+You can use the Yomi formula to install the states and modules you need.
+
+The formula installs the states in [path]``/usr/share/salt-formulas/states``, and configuration files in [path]``/usr/share/yomi``.
+It also stores some pillar examples in [path]``/usr/share/yomi/pillar``.
+
+.Procedure: Installing the Yomi Formula
+
+. On the {productname} Server, at the command prompt, as root, install the formula:
++
+----
+zypper in yomi-formula
+----
+. So that ``salt-master`` can find the pillar, change the
+``pillar_roots`` entry in the configuration file, or use the one provided by the package:
++
+----
+cp -a /usr/share/yomi/pillar.conf /etc/salt/master.d/
+----
+. Restart services:
++
+----
+systemctl restart salt-master.service
+----
+
+== Prepare the Salt Pillar
```

Comment:
This section can be considered the core of the documentation. The user can create the pillar manually, as described here. But because is also using Uyuni, after installing the yomi-formula the user can access to a new set of forms.

Maybe I can prepare some screenshots about how the user can use Uyuni and the forms to create pillars?

Reply:
Sure, thanks :)